kusto/kusto.go

```diff
@@ -294,10 +295,14 @@ func setQueryOptions(ctx context.Context, op errors.Op, query Statement, options
 	return opt, nil
 }
 
-func setMgmtOptions(ctx context.Context, op errors.Op, query Stmt, options ...MgmtOption) (*mgmtOptions, error) {
-	params, err := query.params.toParameters(query.defs)
-	if err != nil {
-		return nil, errors.ES(op, errors.KClientArgs, "QueryValues in the the Stmt were incorrect: %s", err).SetNoRetry()
+func setMgmtOptions(ctx context.Context, op errors.Op, query Statement, options ...MgmtOption) (*mgmtOptions, error) {
+	var params map[string]string
+	if stmt, ok := query.(Stmt); ok {
```

Comment:
Statements support options but not parameters.
Not sure I'm following what's going on here

Reply:
Fixed